README.md

```diff
@@ -14,6 +14,7 @@ There is initial aarch64 support. For more information please check the [README.
 3. Install genisoimage on host `sudo apt install genisoimage`
 4. Install jq on host `sudo apt install jq`
 5. Ensure that docker is installed (docker-ce recent versions, i.e. v15 onwards)
+6. When adding shell tests `tests/integrations/run.bats` execute `make shellcheck` to lint the file
```

Comment:
I think this should not be in the Installation steps. We should have a `dev.md` readme probably to include these steps. I've created an issue for it. In the meantime, let's remove this line. https://github.com/nabla-containers/runnc/issues/80